frontend/www/js/omegaup/components/problem/CollectionDetails.vue

```diff
@@ -1,20 +1,66 @@
 <template>
   <div>
     <h1 class="card-title">{{ title }}</h1>
+    <div class="row">
+      <div class="col col-md-4">
+        <div>
+          <omegaup-problem-filter-tags
+            :tags="collectionTags"
+            :public-tags="publicTags"
+            :selected-tags="selectedTags"
+            @emit-add-tag="addTag"
+            @emit-check="check"
```

Comment:
Al parecer, `check` recibe un parámetro, que es el listado de checks seleccionados, entonces lo tienes que mandar de esta forma:
```suggestion
            @check="(selectedTags) => check(selectedTags)"
```
El `(selectedTags)` que aparece antes de `=>` se está obteniendo del componente donde estás emitiendo la acción.